build.rs

```diff
@@ -1,26 +1,35 @@
 use git2::Repository;
-use std::env;
-use std::fs::{self, File};
-use std::io::Write;
-use std::path::Path;
+use std::{
+    env,
+    error::Error,
+    fs::{self, File},
+    io::Write,
+    path::Path,
+};
 
 fn main() {
     // Set the host target
     println!(
         "cargo:rustc-env=CRATESFYI_HOST_TARGET={}",
-        env::var("TARGET").unwrap()
+        env::var("TARGET").unwrap(),
     );
 
     // Don't rerun anytime a single change is made
-    println!("cargo:rerun-if-changed=templates/style.scss");
+    println!("cargo:rerun-if-changed=templates/style/base.scss");
+    println!("cargo:rerun-if-changed=templates/style/_rustdoc.scss");
+    println!("cargo:rerun-if-changed=templates/style/_vars.scss");
+    println!("cargo:rerun-if-changed=templates/style/_utils.scss");
+    println!("cargo:rerun-if-changed=templates/style/_navbar.scss");
     println!("cargo:rerun-if-changed=templates/menu.js");
     println!("cargo:rerun-if-changed=templates/index.js");
     // TODO: are these right?
     println!("cargo:rerun-if-changed=.git/HEAD");
     println!("cargo:rerun-if-changed=.git/index");
 
     write_git_version();
-    compile_sass();
+    if let Err(sass_err) = compile_sass() {
+        panic!("Error compiling sass: {}", sass_err);
+    }
```

Comment:
```suggestion
    compile_sass().expect("error compiling sass");
```

Reply:
I did this because it makes a better error. Using the `Display` impl allows the error produced to actually be shown in a readable form instead of `being\nlike\nthis\n`

Reply:
Ah right, this uses Display instead of Debug. I wish `expect` would use that by default :/ This change looks fine though.